flixel/addons/display/FlxClothSprite.hx

```diff
@@ -0,0 +1,533 @@
+package flixel.addons.display;
+import flixel.FlxCamera;
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.graphics.FlxGraphic;
+import flixel.graphics.frames.FlxFrame;
+import flixel.graphics.frames.FlxFrame.FlxFrameAngle;
+import flixel.graphics.frames.FlxFrame.FlxFrameType;
+import flixel.math.FlxPoint;
+import flixel.math.FlxRect;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+import flixel.util.FlxSpriteUtil;
+import openfl.display.BitmapData;
+import openfl.display.Graphics;
+import openfl.geom.Point;
+import openfl.Vector;
+
+@:keep @:bitmap("assets/images/logo/default.png")
+private class GraphicDefault extends BitmapData {}
+
+/**
+ * A FlxSprite that draw it's frame in a mesh and behave like a cloth.
+ * 
+ * @author adrianulima
+ */
+class FlxClothSprite extends FlxSprite
+{
+	/**
+	 * An extra speed applied to mesh (in pixels per second). Use to simulate gravity or wind.
+	 */
+	public var gravity(default, null):FlxPoint = FlxPoint.get();
```

Comment:
I like the `meshVelocity` name, I was avoiding `acceleration` to not confuse with the original ones.
I think `gravity` seems more like vertical velocity and `wind` horizontal velocity. So I think `meshVelocity` is realy better.